lib/github.js

```diff
@@ -30,31 +36,28 @@ exports.methods.push({
     }
 });
 
+
 exports.methods.push({
     name: 'github.issues',
     method: function (callback) {
 
-        var weekAgo = new Date();
+        const weekAgo = new Date();
         weekAgo.setDate(weekAgo.getDate() - 7);
 
-        var params = Querystring.encode({
+        const params = Querystring.encode({
             sort: 'created',
             state: 'closed',
             since: weekAgo.toISOString()
         });
 
-        return Utils.download('https://api.github.com/repos/hapijs/hapi/issues?' + params, internals.requestOptions, function (err, issues) {
+        return Utils.download(`https://api.github.com/repos/hapijs/hapi/issues?${params}`, internals.requestOptions, (err, issues) => {
 
             if (err) {
                 return callback(err);
             }
 
-            issues = issues.filter(function (issue) {
-
-                return !issue.hasOwnProperty('pull_request');
-            });
-
-            callback(null, issues);
+            issues = issues.filter((issue) => !issue.pull_request);
```

Comment:
It's not the same meaning but can you see any instance where this will produce incorrect behaviour?